pycm/pycm_util.py

```diff
@@ -775,9 +775,37 @@ def inner_function(*args, **kwargs):
         :type args: list
         :param kwargs: keyword arguments
         :type kwargs: dict
-        :return: None
+        :return: modified function result
         """
         if args[0].metrics_off:
             raise pycmMatrixError(METRICS_OFF_ERROR)
         return func(*args, **kwargs)
     return inner_function
+
+
+def deprecated(func):
+    """
+    Send a warning regarding decorated function's deprecation.
```

Comment:
I suggest to change this to:

“ Send a warning regarding function's deprecation.”